### PR TITLE
Nexus deploy

### PR DESCRIPTION
**Pull request by jhowarth**

### CODE REVIEW

**Comment by jhowarth** [comment]
@reset @ivey @bluepojo

**Comment by reset** [comment]
@jhowarth this looks solid to me :+1:

File: providers/deploy.rb [hunk 1]
```diff
@@ -197,26 +209,59 @@ def write_completion_token
...
+      owner new_resource.owner
+      group new_resource.group
+      backup false
+
+      action :create
+    end
+  end
+
+  def retrieve_from_nexus
+    chef_gem 'nexus_cli'
```
Comment (reset) [inline providers/deploy.rb]: I would lock this version to 0.0.3 just in case

Comment (reset) [inline providers/deploy.rb]: actually lock to 0.0.4, @KAllan357 just pushed a new version which fixes some broken things

**Pull request merged**